variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "enabled" {
+  type    = bool
+  default = true
+}
+
+variable "critical_threshold" {
+  type    = number
+  default = 1
+}
+
+variable "warning_threshold" {
+  type    = number
+  default = null
+}
+
+variable "ok_threshold" {
+  type    = number
+  default = 1
+}
+
+variable "no_data_timeframe" {
+  description = "Minutes of no data coming in. Scrape interval is approx 10 minutes"
+  type        = number
+  # Disabled because the measurements are data-blank moment-data-blank moment...
```

Comment:
Could you clarify this? It took me a couple of reads to understand this.

I am not sure I understand the issue either. Assuming that the blank moments are constant (10 minutes), shouldn't we just make sure that the value is high enough that it always covers a data point?

Reply:
I've updated the description. But I'm not interested in alerts when DD's scraping is kaput